web_src/js/markup/content.js

```diff
@@ -1,9 +1,11 @@
 import {renderMermaid} from './mermaid.js';
+import {renderCodeCopy} from './codecopy.js';
 import {initMarkupTasklist} from './tasklist.js';
 
 // code that runs for all markup content
 export function initMarkupContent() {
-  const _promise = renderMermaid(document.querySelectorAll('code.language-mermaid'));
+  renderMermaid();
+  renderCodeCopy();
```

Comment:
If there is no `mermaid`, should the `renderCodeCopy` be executed?

If `renderCodeCopy` depends on `mermaid`, it's better to skip `renderCodeCopy`, otherwise `document.querySelectorAll` still affects page loading performance.

Reply:
There is no interdependency, both render on pages that have `.markup`. I guess right now executing two `querySelectorAll` won't cost much and if the number of features grows more, we could add a query for `.markup` to quickly skip them all but I don't see that necessary right now.